src/sentry/tsdb/snuba.py

```diff
@@ -230,7 +230,7 @@ def get_data(
             keys = list(set(map(lambda x: int(x), keys)))
 
         # 10s is the only rollup under an hour that we support
-        if rollup and rollup == 10 and model in self.lower_rollup_query_settings.keys():
+        if rollup and rollup == 10 and model in self.lower_rollup_query_settings:
```

Comment:
I removed a few things like this too. It's unnecessary (and actually less efficient) to access .keys when checking for membership.